content/installation/ruby/server/Passenger.md

```diff
@@ -0,0 +1,66 @@
+<!--
+title: "Installing with Passenger"
+description: "Recommendations for installing in applications run by Passenger"
+tags: "ruby agent installation Passenger"
+-->
+
+## Inability to Start the Application
+
+### Cause - Application Timeout
+
+The most common issue faced when integrating with an application started and monitored by Passenger is the increase in
```

Comment:
@hmcginnis Is this article intended to be used primarily as troubleshooting?

Reply:
it's a combination of trouble shooting and installation instructions when the agent is installed in applications managed by Passenger